core/server/index.js

```diff
@@ -37,11 +37,12 @@ var debug = require('ghost-ignition').debug('boot:init'),
 function init() {
     debug('Init Start...');
 
-    var ghostServer, parentApp;
+    var ghostServer, parentApp, i18nCore;
 
-    // Initialize Internationalization
-    i18n.init();
-    debug('I18n done');
+    // Initialize default internationalization, just for core now
+    // (settings for language and theme not yet available here)
+    i18n.init(true, false);
```

Comment:
The `init` function is called too often.

e.g. if i reset my database and serve `/ghost`, i am getting two warnings - probably because `init` is called [here](https://github.com/TryGhost/Ghost/pull/8437/files#diff-12a8dbd78d369667d9a6160c856477dfR232) on **every** settings change.

> [2017-11-21 21:34:47] WARN Theme's file assets/locales/en.json not found! Falling back to default English locale.
> [2017-11-21 21:34:47] WARN Theme's file assets/locales/en.json not found! Falling back to default English locale.

I think we need a cleaner differentiation between core initialisation and theme initialisation and we should take care that it's not called in too many places, only if really required.

Reply:
Maybe we can differentiate between `i18n.init`, which initialises the `en` core translation and between `i18n.loadThemeTranslations`.

Reply:
Done this change with yesterday's commit:
[i18n] Function `I18n.loadThemeTranslations` separated from `I18n.init`